content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -10,319 +10,338 @@ further_reading:
+
+### Resource Detectors
+
+If provided by the language SDK, it can be useful to attach container information as resource attributes. For example, in Go this can be done using the [`WithContainer()`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/resource#WithContainer) resource option.
+
+### Unified Service Tagging
+
+Unified service tagging ties Datadog telemetry together through using three essential tags that define:
+* The service name
+* The deployment environment
+* The service version
 
-### Docker
+The application should set these using the OpenTelemetry semantic conventions: `service.name`, `deployment.environment` and `service.version`.
 
-Run an Opentelemetry Collector container to receive traces either from the [installed host](#receive-traces-from-host), or from [other containers](#receive-traces-from-other-containers).
+## Using Docker
+
+Run an Opentelemetry Collector container to receive traces either from [localhost](#receive-traces-from-host), or from [other containers](#receive-traces-from-other-containers).
 
 <div class="alert alert-info">
 The latest tag of the OpenTelemetry Collector Contrib distro <a href="https://github.com/open-telemetry/opentelemetry-collector-releases/issues/73">is not updated on every release</a>.
 Pin the Collector to the latest version to pick up the latest changes.
 </div>
 
-#### Receive traces from host
+### Receive traces from localhost
 
-1. Create a `otel_collector_config.yaml` file. [Here is an example template](#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's OTLP receiver and the Datadog exporter.
+Follow these steps if you wish to run the OpenTelemetry Collector as a Docker image and receive traces from the same host:
 
-2. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib:latest`][10].
+1. Create a `collector.yaml` file. You may use the [example template above](#configuring-the-datadog-exporter) to get started.
 
-3.  Determine which ports to open on your container. OpenTelemetry traces are sent to the OpenTelemetry Collector over TCP or UDP on several ports, which must be exposed on the container. By default, traces are sent over OTLP/gRPC on port `4317`, but common protocols and their ports include:
+2. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib:<VERSION>`][10].
 
-      - Zipkin/HTTP on port `9411`
-      - Jaeger/gRPC on port `14250`
-      - Jaeger/HTTP on port `14268`
-      - Jaeger/Compact on port (UDP) `6831`
-      - OTLP/gRPC on port `4317`
-      - OTLP/HTTP on port `4318`
+3. Determine which ports to open on your container. OpenTelemetry traces are sent to the OpenTelemetry Collector over HTTP on two ports, depending on whether you are using gRPC or not. These ports must be exposed on the container. By default, traces are sent over gRPC on port 4317.
```

Comment:
```suggestion
3. Determine which ports to open on your container so that OpenTelemetry traces are sent to the OpenTelemetry Collector. By default, traces are sent over gRPC on port 4317. If you don't use gRPC, use port 4138.
```
Right?